config/locales/client.en.yml

```diff
@@ -4299,13 +4299,11 @@ en:
     pause_notifications:
       title: "Pause notifications for..."
       label: "Pause notifications"
-      remaining: "%{remaining} remaining"
       options:
         half_hour: "30 minutes"
         one_hour: "1 hour"
         two_hours: "2 hours"
         tomorrow: "Until tomorrow"
-        custom: "Custom"
```

Comment:
@AndrewPrigorshnev Is this used somewhere? I couldn't find it.

Reply:
I also don't see any usages. I think we're safe to delete it.